Luna/Luna/AddPeriodViewController.swift

```diff
@@ -104,6 +92,58 @@ class AddPeriodViewController: UIViewController
         setLabelDates()
     }
     
+    fileprivate func onSavePressed()
+    {
+        let canSave = checkDate()
+        
+        if canSave
+        {
+            createPeriodObject()
+            delegate?.onLastCycleChange()
+            delegate?.onDismissAddPeriod()
+        }
+    }
+    
+    fileprivate func checkDate() -> Bool
+    {
+        
+        let selectDate = addPeriodViewModel.Start
+        let endDate = addPeriodViewModel.End
+     
+        let order = Calendar.current.compare(selectDate, to: endDate, toGranularity: .day)
+        
+        switch order
+        {
+            case .orderedDescending:
+                let alert = UIAlertController( title: Constants.InterfaceBuilderStrings.pastDateTitle,
+                                               message: Constants.InterfaceBuilderStrings.pastDateMessage,
+                                               preferredStyle: .alert )
+                alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: { (action: UIAlertAction!) in
```

Comment:
Also, this can be pulled out into a separate method. From the top, the method name is `checkDate()`. 

This isn't very descriptive. What does this do? What is it checking? Why is it checking _and_ presenting an alert? that's too much responsibility for this method. Maybe we can pull it out.